src/test/resources/org/broadinstitute/hellbender/tools/copynumber/utils/combine-segment-breakpoints-different-annotation-headers-with-legacy-header.tsv

```diff
@@ -1,4 +1,6 @@
-#SAMPLE_NAME=SAMPLE2
+@HD	VN:1.5
```

Comment:
I have removed the `-with-legacy-headers` from the test file names.

Currently, since this is being driven by a config file (that contains CONTIG, START, and END), the input files are meant to represent files either generated by the CNV pipeline.  

These files could already conceivably have been generated by `CombineSegmentBreakpoints`.  

Are you asking me to test input files with different config file formats?

Reply:
My point was that your test files should primarily reflect actual inputs expected by the tool.

Now that you've changed over to config files that identify the column numbers of the locatable fields, I would not expect to see this file as a typical input---it's a mishmash of CNV 1.0/ReCapSeg-style columns, modified `CONTIG\tSTART\tEND` column headers, and a CNV-collections-style SAM header.

Reply:
I was just using `Segment_Mean` and `Segment_Call` (CNV 1.0 style columns) as two example annotations.  While not likely an exact input (unless I was comparing CNV 1.0 to CNV ModeledSegments), the header can really be CONTIG, START, END + anything.  The only requirement for the columns for this tool is to have CONTIG, START, and END.

Reply:
I think that it is somewhat confusing to use this as test input if we'd never really see it in the real world.  As a developer who might have to maintain the tests that use this input, it's nice to be able to understand what the origin of the input might be (in case I have to modify it or come up with new inputs).  The name of the test file (*-different-annotation-headers-with-legacy-header.tsv) doesn't really provide enough clues in that regard, either.

Why not just use an unmodified ReCapSeg or CNV 1.0 seg file as test input?

Reply:
Using fake data, I have included test files from JaBbA, UCSC, ReCapSeg (which is like CNV 1.0), PCAWG consensus files, and an Oncotator annotated CNV output.

Done.